docs/concepts/gardenlet.md

```diff
@@ -1,119 +1,137 @@
 # Gardenlet
 
-Right from the beginning of the Gardener Project we started implementing the [operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/):
-We have a custom controller-manager that acts on our own custom resources.
-Now, when you start thinking about the [Gardener architecture](https://github.com/gardener/documentation/wiki/Architecture), you will recognize some interesting similarity with respect to the Kubernetes architecture:
-Shoot clusters can be compared with pods, and seed clusters can be seen as worker nodes.
-Guided by this observation we introduced the **gardener-scheduler** ([#356](https://github.com/gardener/gardener/issues/356)).
-Its main task is to find an appropriate seed cluster to host the control-plane for newly ordered clusters, similar to how the kube-scheduler finds an appropriate node for newly created pods.
-By providing multiple seed clusters for a region (or provider) and distributing the workload, we reduce the blast-radius of potential hick-ups as well.
-
-Yet, there was still a significant difference between the Kubernetes and the Gardener architectures:
-Kubernetes runs a primary "agent" on every node, the kubelet, which is mainly responsible for managing pods and containers on its particular node.
-Gardener used its single controller-manager which was responsible for all shoot clusters on all seed clusters, and it was performing its reconciliation loops centrally from the garden cluster.
-
-While this works well at scale for thousands of clusters today, our goal is to enable true scalability following the Kubernetes principles (beyond the capacity of a single controller-manager):
-We have now worked on distributing the logic (or the Gardener operator) into the seed cluster and introduced a corresponding component, adequately named the **Gardenlet**.
-It is Gardener's primary "agent" on every seed cluster and is only responsible for shoot clusters located in its particular seed cluster.
+Gardener is implemented using the [operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/): It uses a custom `controller-manager` that acts on our own custom resources, and applies Kubernetes principles to manage clusters instead of containers. Following this analogy, you can recognize components of the Gardener architecture as well-known Kubernetes components, for example, shoot clusters can be compared with pods, and seed clusters can be seen as worker nodes.
```

Comment:
```suggestion
Gardener is implemented using the [operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/): It uses custom controllers that act on our own custom resources, and apply Kubernetes principles to manage clusters instead of containers. Following this analogy, you can recognize components of the Gardener architecture as well-known Kubernetes components, for example, shoot clusters can be compared with pods, and seed clusters can be seen as worker nodes.
```